app/controllers/events/view/export.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+
+const { Controller, run, $ } = Ember;
+export default Controller.extend({
+  eventExportStatus : '',
+  eventDownloadUrl  : '',
+  actions           : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('model.exportData.dataImage'),
+        video    : this.get('model.exportData.dataVideo'),
+        audio    : this.get('model.exportData.dataAudio'),
+        document : this.get('model.exportData.dataDocument')
+      };
+      let exportUrl = `/events/${this.get('model').currentEventID}/export/json`;
+      this.get('loader')
+        .post(exportUrl, payload)
+        .then(data => {
+          let download_url = data.task_url.split('/').splice(2, 4).join('/');
```

Comment:
I have updated the loader service's config handling to be a bit more sane (in https://github.com/fossasia/open-event-frontend/commit/39c9bae55e50052bd655bc70000c68d328e49959) . Now just pass in a config object with `withoutPrefix: true` and you can pass in the task_url url directly. Without having to play with the string in anyway.